### PR TITLE
Case studies

### LINKED ISSUE #4404: Create a section on the homepage for Knative Case Studies
@xtreme-sameer-vohra has context on this, he will follow up with a PR

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #4404 

cc @xtreme-sameer-vohra @omerbensaadon

### CODE REVIEW

**Comment by csantanapr** [comment]
latest preview is here https://deploy-preview-4406--knative.netlify.app/development/

File: config/nav.yml [hunk 1]
```diff
@@ -273,4 +273,9 @@ nav:
         - Eventing: reference/api/eventing-api.md
       - Concepts:
         - Duck types: reference/concepts/duck-typing.md
+    - Case Studies:
+      - deepc: about/case-studies/deepc.md
+      - outfit7: about/case-studies/outfit7.md
+      - puppet: about/case-studies/puppet.md
```
Comment (omerbensaadon) [inline config/nav.yml:279]:
```suggestion
      - Outfit7: about/case-studies/outfit7.md
      - Puppet: about/case-studies/puppet.md
```

**Comment by csantanapr** [comment]
/unhold
/assign @omerbensaadon 

This is ready now 🚀 1.0

**Comment by omerbensaadon** [comment]
/hold 
uhhh I don't know what's going on here, but the nav is being very weird... 

Click Blog and the nav changes?

**Comment by omerbensaadon** [comment]
@googlebot I consent

**Comment by omerbensaadon** [comment]
@googlebot I consent.

**Comment by omerbensaadon** [comment]
Fixed some nav issues for ya!

/lgtm

**Comment by csantanapr** [comment]
/unhold

thanks @omerbensaadon for merging the blog

**Pull request merged**